app/src/main/java/org/fossasia/openevent/app/data/models/SimpleModel.java

```diff
@@ -0,0 +1,46 @@
+package org.fossasia.openevent.app.data.models;
+
+import com.raizlabs.android.dbflow.annotation.PrimaryKey;
+import com.raizlabs.android.dbflow.annotation.Table;
+import com.raizlabs.android.dbflow.structure.BaseModel;
+
+import org.fossasia.openevent.app.data.db.configuration.OrgaDatabase;
+
+@Table(database = OrgaDatabase.class, allFields = true)
+public class SimpleModel extends BaseModel {
+
+    @PrimaryKey
+    public long id;
+
+    public String name;
+    public String description;
+
+    SimpleModel() {}
```

Comment:
It is created to test that DBFlow configuration is working, not to be used by outer package